tests/kvmtest.sh

```diff
@@ -268,7 +270,7 @@ test_multi_asic_t1_lag() {
 
     pushd $SONIC_MGMT_DIR/tests
     # TODO: Remove disable of loganaler and sanity check once multi-asic testbed is stable.
-    ./run_tests.sh $RUNTEST_CLI_COMMON_OPTS -c "$tests" -p logs/$tgname -e --disable_loganalyzer -e --skip_sanity -u
+    ./run_tests.sh $MULTI_ASIC_CLI_OPTIONS -c "$tests" -p logs/$tgname -e --disable_loganalyzer -e -u
```

Comment:
should remove "-e --skip_sanity", -e is not removed.

Reply:
thanks for pointing out, fixed.